Documentation/policy/language.rst

```diff
@@ -745,6 +745,8 @@ If a layer 4 rule is specified in the policy, and a similar layer 4 rule
 with layer 7 rules is also specified, then the layer 7 portions of the
 latter rule will have no effect.
 
+To see some L7 Network Policy exameples in action check :ref:`gs_http`
```

Comment:
It feels a bit weird to point from the Policy documentation to the getting started guide. I'd expect users to come on this page if they are looking to answer some specific question and already have the basics working.

Reply:
Yes, I see your point. In my case, I arrived to this page by googling "Cilium L7 Network Policy" and I was unaware until days later (by chance) that there were useful examples in the Getting Started guide for L7 Network Policy. I think a link here might be useful for other "unstructured" doc readers that don't follow the logical way of reading documentation. What if I add something like: "If you haven't done it yet, please have a look at our Getting Started guide which provides comprehensive L7 Network Policy examples :ref:`gs_http`"?

Reply:
I don't feel strongly against this, but then, why add this link for the L7 policies and not for others (e.g., L3-L4 or host policies)? Maybe another way to address this is: what did you find in the getting started guide (GSG) that you didn't in this page? Does it make sense to add this missing information here? If not, maybe we can point the readers to that specific info. rather than have a general link to the GSG?

Reply:
I liked the fact that there was a hands-on example through which I could understand more stuff about L7 Network Policy because I was seeing it in action, that's why I am pointing to that example but not the whole GSG. Copying stuff from there over here does not make sense to me as we would be duplicating. Adding a link for L3/L4 or host policies would also make sense to me, I could add that. Anyway, I am also not feeling strong about this. Personally, I think this would help, but maybe it is only me :P. If this violates the doc rules or it will complicate things, then I can just drop it, no worries

Reply:
As I hear no other opinion regarding this, I'll drop it from this PR. We can always add it in another PR in the future :)